src/StratisMasternodeDashboard/Controllers/SidechainNodeController.cs

```diff
@@ -158,6 +161,78 @@ public async Task<IActionResult> VoteSDAProposal([FromBody] SDAVoteModel sDAVote
             } while (true);
         }
 
+        //PendingPoll
+        [Ajax]
+        [Route("Getpolls")]
+        [HttpGet]
+        public async Task<IActionResult> Getpolls()
+        {
+            List<PendingPoll> result;
+
+            result = await UpdatePolls().ConfigureAwait(false);
+            int federationmemberCount = await UpdateFederationMemberCount().ConfigureAwait(false);
+            return PartialView("Partials/ProofOfAuthority", new Vote() { Polls = result, FederationMemberCount = federationmemberCount });
+
+        }
+
+        private async Task<List<PendingPoll>> UpdatePolls()
```

Comment:
I think lets name this method properly, lets call it `UpdateWhitelistedHashPolls`.